### PR TITLE
py : cleanup the code

### PR DESCRIPTION
**Pull request by prusnak**
- use f-strings where possible
- drop first param of encode/decode functions since "utf-8" is the default

### CODE REVIEW

**Comment by prusnak** [comment]
Rebased on top of current master

File: convert-pth-to-ggml.py [hunk 1]
```diff
@@ -268,7 +268,25 @@ def main():
             process_and_write_variables(fout, model, ftype, part_id, n_parts)
             del model
 
+<<<<<<< HEAD
```
Comment (j-f1) [inline convert-pth-to-ggml.py:271]: Merge conflict?

**Review by j-f1 — CHANGES_REQUESTED** [review]

File: convert-pth-to-ggml.py (hunk 1 above)
Comment (prusnak) [reply convert-pth-to-ggml.py]: Right. Fixed in 0a98613d9ee983c691534bbbb58f7d8a899c0494

**Review by j-f1 — APPROVED** [review]

**Review by j-f1 — APPROVED** [review]

**Comment by prusnak** [comment]
Rebased after https://github.com/ggerganov/llama.cpp/pull/640 has been merged

**Review by j-f1 — APPROVED** [review]

**Pull request merged**